test/compose.clone.js

```diff
@@ -5,7 +5,11 @@ const metatests = require('metatests');
 
 metatests.test('async functions composition clone', test => {
   const data = { test: 'data' };
-  const expectedData = { test: 'data', data1: 'data 1', data2: 'data 2' };
+  const expectedData = {
+    test: 'data',
+    data1: 'data 1',
+    data2: 'data 2',
+  };
```

Comment:
There is no need to make those objects multiline, out rules don't prohibit such usage.